### DISCUSSION TITLE
ESP32+ILI9341 DMA not working?

### DISCUSSION THREAD

**Discussion by Krystian0** [Bodmer/TFT_eSPI#1513] [Q&A - General]
Is pushImageDMA supposed to work on esp32? It doesn't for me, when i do pushImage with same sprite, it works.

This example doesn't work, only part working is fillScreen
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/DMA%20test/Bouncy_Circles/Bouncy_Circles.ino

MISO is not connected, could this be the reason? I don't need to read screen contents so i omitted it.

**Comment by Bodmer** [comment]
Uncomment this line in your copy of the sketch.
https://github.com/Bodmer/TFT_eSPI/blob/master/examples/DMA%20test/Bouncy_Circles/Bouncy_Circles.ino#L93
